src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Machine> {
+
+    @Override
+    public List<Machine> get() {
```

Comment:
let's fix type according to the task description 
![image](https://user-images.githubusercontent.com/58385506/134124658-2c7e7bba-ebc4-4a52-ace1-7f84fe152ab3.png)